### ISSUE TITLE
Publish to NuGet

### ISSUE THREAD

**Issue by adamdriscoll** [PowerShell/PowerShellEditorServices#41]
I know this is likely planned for this library but I figure it should be documented! 

Would love to see this on the NuGet gallery so I can just pull it into my projects.

**Comment by adamdriscoll** [comment]
I see there is already a NuSpec file so I figure it's just the publishing that needs to happen. :D

**Comment by daviwil** [comment]
Yep, I just need to publish them but ran out of time dealing with a couple last minute details.  Re-opening to track this :)

**Comment by daviwil** [comment]
Forgot to close this one!

**Comment by rjmholt** [comment]
It looks like we haven't done this in a while ([here](https://www.nuget.org/packages/Microsoft.PowerShell.EditorServices/)'s the last publication).

With renewed interest in getting API documentation up to date (and ideally publishing it on docs.microsoft.com), I've reopened this issue to track nuget package generation in our new build infrastructure and to make sure nuget publication becomes part of our regular release process.

**Comment by andyleejordan** [comment]
Not planned — PSES ships as a PowerShell module rather than a NuGet package. Closing as obsolete.

---
_Drafted by GitHub Copilot (Claude Opus 4.8) at @andyleejordan's request._

**Comment by JustinGrote** [comment]
Technically a PowerShell module *is* a NuGet Package, so it could be closed completed :P
